script.js

```diff
@@ -1,12 +1,22 @@
 const stuffQuizProxiedURL = 'https://corsproxy.io/?' + encodeURIComponent('https://www.stuff.co.nz/_json/national/quizzes?limit=99');
+import { initializeApp } from "https://www.gstatic.com/firebasejs/10.8.0/firebase-app.js";
+import { addDoc, collection, getDocs, getFirestore, onSnapshot, query, updateDoc, where } from "https://www.gstatic.com/firebasejs/10.8.0/firebase-firestore.js";
 
-window.onload = async function() {
-  const stories = await fetchQuizzes();
-
-  renderQuizLinks(document.getElementById("three-strikes"), stories.threeStrikes);
-  renderQuizLinks(document.getElementById("hard-words"), stories.hardWords);
-  renderQuizLinks(document.getElementById("quizzes"), stories.quizzes);
+const stories = await fetchQuizzes();
+const firebaseConfig = {
+  apiKey: "AIzaSyAoYRHH_PI7IPE9zwDiP4tfeGVwHFNhBUU",
+  authDomain: "quizzaroo-f98dc.firebaseapp.com",
+  projectId: "quizzaroo-f98dc",
+  storageBucket: "quizzaroo-f98dc.appspot.com",
+  messagingSenderId: "728917953883",
+  appId: "1:728917953883:web:66fc2f6eb3cccd7c7cde7e"
```

Comment:
Is this stuff safe to be within the frontend?

Does firebase protect from someone else taking these credentials for usage?

Reply:
Yep, it's all good. This is a public key thingy. [Reference](https://firebase.google.com/docs/projects/api-keys).